src/ServicePulse.Host/vue/src/components/monitoring/EndpointList.vue

```diff
@@ -1,10 +1,10 @@
-﻿<script setup>
+﻿<script setup lang="ts">
 import { ref } from "vue";
 import SortableColumn from "../../components/SortableColumn.vue";
 import EndpointListRow from "./EndpointListRow.vue";
-import { useMonitoringStore } from "../../stores/MonitoringStore";
+import { type MonitoringStore, useMonitoringStore } from "@/stores/MonitoringStore";
 
-const monitoringStore = useMonitoringStore();
+const monitoringStore: MonitoringStore = useMonitoringStore();
```

Comment:
That is weird, isn't the stored typed?

Reply:
That is correct. The store type is useful when passing the store to a function. When assigned to a const, like in a component, adding the type is not required.

Reply:
i figured this out yesterday but haven't made the change yet :)